rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(rosidl_generator_cpp)
 
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
+find_package(rosidl_generator_c REQUIRED)
```

Comment:
Since the header of this package transitively exposes the header from `rosidl_generator_c` does it need to be an export dependency?

Reply:
I agree with you, but the odd thing is that it was working without it (I would have expected the builds to fail to find the right header file).  In any case, I've now changed this to have an `ament_export_dependency()`.